documentation/asciidoc/topics/proc_batching_configmap.adoc

```diff
@@ -0,0 +1,54 @@
+[id='batching-configmap_{context}']
+= Running Batch Scripts with Configuration Files
+Use a `ConfigMap` for batch operations if you want to use additional files such as an `infinispan.xml` cache configuration.
+
+.Procedure
+
+. Create a batch script named `batch` that contains all the CLI operations you want to run, for example:
++
+[source,options="nowrap",subs=attributes+]
+----
+create cache --file=mycache.xml mycache
+put --cache=mycache hello world
+put --cache=mycache hola mundo
+disconnect
```

Comment:
Do we really need `disconnect` at the end?

Reply:
hmm. yeah I think we can remove this.